packages/sync-repo-settings/src/sync-repo-settings.ts

```diff
@@ -193,17 +206,34 @@ async function updateRepoTeams(repo: Repo, context: Context) {
   console.log(`Update team access for ${repo.repo}`);
   const [owner, name] = repo.repo.split('/');
   const teamsToAdd = getRepoTeams(repo.language);
-  await Promise.all(
-    teamsToAdd.map(membership => {
-      return context.github.teams.addOrUpdateRepoInOrg({
-        team_slug: membership.slug,
-        owner,
-        org: owner,
-        permission: membership.permission as 'push',
-        repo: name,
-      });
-    })
-  );
+  try {
+    await Promise.all(
+      teamsToAdd.map(membership => {
+        return context.github.teams.addOrUpdateRepoInOrg({
+          team_slug: membership.slug,
+          owner,
+          org: owner,
+          permission: membership.permission as 'push',
+          repo: name,
+        });
+      })
+    );
+  } catch (err) {
+    const knownErrors = [
+      401, // bot does not have permission to access this repository.
+      404, // team being added does not exist on repo.
```

Comment:
Again, interesting!  I'd love to know which teams we're missing that's creating the problem and go address it.  This brings up two issues:
- Did you notice in the logs which ones came up?  
- On an ongoing basis, what would be the right way to look for this situation and react to it?  Or by swallowing the error, are we saying "we are ok never addressing this"?